compiler/optimizer/InductionVariable.cpp

```diff
@@ -689,7 +689,7 @@ int32_t TR_LoopStrider::detectCanonicalizedPredictableLoops(TR_Structure *loopSt
                   }
 
                ListIterator<TR::Block> bi(&exitBlocks);
-               vcount_t visitCount = comp()->incVisitCount();
+               visitCount = comp()->incVisitCount();
```

Comment:
I don't think it is - I think in this case this was previously local and only and so the dominant definition was either 0 or set in an if higher up the function. the thing that makes this safe is that the local visitcount is not actually used directly. All the helpers called always ask comp()->getVisitCount() and so the key thing here is the inc. The local value is not used downstream from what I can see.